### PR TITLE
Add rules and stuff

### PR DESCRIPTION
**Pull request by jessejanderson**

### CODE REVIEW

File: test/pco_api/people/list/rule_test.exs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    record_without_link |> Rule.get
+  end
+
+  test ".get gets rules by list id", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/lists/1/rules" == conn.request_path
+      Plug.Conn.resp(conn, 200, Fixture.read("rules.json"))
+    end
+    record_without_link |> Rule.get
+  end
```
Comment (geolessel) [inline test/pco_api/people/list/rule_test.exs]: What is the difference between this one and the one above it?

File: test/pco_api/people/list/rule_test.exs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+  test ".get gets rules by rule id", %{bypass: bypass} do
+    Bypass.expect bypass, fn conn ->
+      assert "/people/v2/lists/1/rules/1" == conn.request_path
+      Plug.Conn.resp(conn, 200, Fixture.read("rules.json"))
+    end
+    record_without_link |> Rule.get(1)
+  end
+
+  test ".conditions gets a list of conditions with an conditions link", %{bypass: bypass} do
```
Comment (geolessel) [inline test/pco_api/people/list/rule_test.exs]: "with a"

File: test/pco_api/people/list/rule_test.exs (hunk 1 above)
Comment (jessejanderson) [inline test/pco_api/people/list/rule_test.exs]: Hmmm, I copied it from somewhere else, I'll investigate.

**Comment by geolessel** [comment]
👍

**Pull request merged**